specification/security/put_role/SecurityPutRoleRequest.ts

```diff
@@ -38,12 +38,44 @@ export interface Request extends RequestBase {
     refresh?: Refresh
   }
   body?: {
+    /**
+     * A list of application privilege entries.
+     */
     applications?: ApplicationPrivileges[]
+    /**
+     * A list of cluster privileges. These privileges define the cluster-level
+     * actions for users with this role.
```

Comment:
Drive-by comment:
Multi-line comments insert escaped line breaks (`\n`) into the generated `schema.json`. I've been avoiding them, but it may not matter. @delvedor or @sethmlarson may be able to say.

We probably wouldn't want to insert line breaks into any docs we generate from this tho.

Reply:
Thanks @jrodewig, that's good feedback. I inspected the generated `schema.json` and see the line breaks. I removed them in 869a5de53ed9c8c48a17073a1ee2b7d77c6e8642

Reply:
I think this should not be happening, we should be able to write multiline descriptions, but the resulting JSON should not contain line breaks. I'll send a pr to address this :)